### PR TITLE
Feature/logback config

### PR DESCRIPTION
**Pull request by Samurus**

### CODE REVIEW

File: src/main/java/ch/softridge/cardmarket/autopricing/domain/service/PriceService.java [hunk 1]
```diff
@@ -27,13 +27,13 @@
   @Autowired
   private MkmService mkmService;
 
-  @Value("${pricer.uri}")
+  @Value("${pricer.uri:http://localhost:18081/prod/pricer/}")
```
Comment (KZellweger) [inline src/main/java/ch/softridge/cardmarket/autopricing/domain/service/PriceService.java:30]: Default values belong into the corresponding application-{profile}.yml

File: src/main/resources/logback-spring.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
```
Comment (KZellweger) [inline src/main/resources/logback-spring.xml:3]: Conflicts with https://github.com/KZellweger/mkmWebClient/blob/develop/src/main/resources/logback-spring.xml

**Review by KZellweger — COMMENTED** [review]
In general the Structure of the project changes drastic at the moment. 
The Direct integration of the pricer module has been removed for the moment, as i want first get to a clean architecture. 
see https://github.com/KZellweger/mkmWebClient/tree/feature/architecture-cleanup for a generell overview.  (Not runnable at the Moment)

**Comment by Samurus** [comment]
With that kind of refactoring it makes no sense to work on master :) Whats the first MVP?

**Comment by KZellweger** [comment]
Well there is not a dedicated MVP. 
I drew a small architecture diagram that I can post in this Repo's Wiki. (Probably somewhat tomorrow evening, as I'm pretty busy today) 
In summary first task is to re-implement the cardmarket4J package. First because it has circular dependencies within itself and second because there is not really an errorhandling present that is sufficient for such a key-component. 
While doing this ill also integrate it within the current entity-model so that there will be just one and not a million Mappers. (Was causing a headache) 
Thats what im currently working on. (Latest changes not pushed to Github i think) 

Second thing to do which could be done independent, is to setup an Infrastructure for BatchJobs (using Spring Batch). With those i want to handle big Data operations such as syncing Products Database with Cardmarket, as well later on automatically Pricing calculations and updates

**Pull request closed without merging**